pallets/loans-ref/src/weights.rs

```diff
@@ -0,0 +1,57 @@
+//! Autogenerated weights for pallet_loans_ref
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-03-09, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("development"), DB CACHE: 1024
+
+// Executed Command:
+// target/release/centrifuge-chain
+// benchmark
+// pallet
+// --chain=development
+// --steps=50
+// --repeat=20
+// --pallet=pallet-loans-ref
+// --extrinsic=*
+// --execution=wasm
+// --wasm-execution=compiled
+// --heap-pages=4096
+// --template=./scripts/frame-weight-template.hbs
+// --output=./pallets/loans-ref/src/weights.rs
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{
+	traits::Get,
+	weights::{constants::RocksDbWeight, Weight},
+};
+use sp_std::marker::PhantomData;
+
+/// Weight functions needed for pallet_loans_ref.
+pub trait WeightInfo {
+	fn update_portfolio_valuation(n: u32, m: u32) -> Weight;
```

Comment:
Pretty sure we do not need the `m` (the MaxRateCount`) parameter because we use it later always with the max value.

Reply:
Answered below by Branan